### DISCUSSION TITLE
特定网络条件下自动切换策略

### DISCUSSION THREAD

**Discussion by loyle5** [MetaCubeX/ClashMetaForAndroid#372] [Q&A]
有没有详细的策略规则说明？
想要实现特定wifi下自动切换策略，有没有实现的方法？

**Comment by loveyu** [comment]
希望可以在 Automation 中增加配置切换功能，这样就完美了
然后可以通过第三方工具自动化处理，实现起来也比较容易，就是不知道作者是否考虑
